README.md

```diff
@@ -47,32 +47,48 @@
   </a>
 </p>
 
-In essence Vocabulary is a collection of UI components, which are parts of 
+In essence Vocabulary is a collection of UI components, which are parts of
 web sites and web apps that have significance as being individual units that
-can be mixed, matched, styled and put together to quickly and effectively 
+can be mixed, matched, styled and put together to quickly and effectively
 develop consistent applications.
 
-
 ### Availability
 
 Vocabulary is available as
+
 - [stylesheets](https://github.com/creativecommons/vocabulary)
 - Vue components _(this repo)_
 
-
 ### Demo
 
 The showcase of the true extent of Vocabulary can be found in the
-**[live styleguide](https://vue-vocabulary.netlify.com/styleguide)**
+**[live styleguide](https://cc-vue-vocabulary.netlify.com/styleguide)**
 and the
-**[interactive storybook](https://vue-vocabulary.netlify.com/storybook)**
+**[interactive storybook](https://cc-vue-vocabulary.netlify.com/storybook)**
 for the library, showcasing the components therein and variants thereof, on GitHub
 Pages. Search for a particular component using the search field or just explore
 the collection of components. If you find one that catches your fancy, play with
 it on the playground and tweak its knobs.
 
-If it works for you, just pick the code and use it in your own application. If 
-it doesn't and you think more people would love to have this feature open an 
+<p align="center">
```

Comment:
Let's not center this. Would allow us to use Markdown `#` and `##` instead of HTML. Also both screenshots should be inside the demo section.

Reply:
Done

Reply:
It's still `<p align="center"><h2 align="center">...` which means this was **not** done.